bitnami/kubeapps/values.yaml

```diff
@@ -389,13 +447,15 @@ frontend:
     ## @param frontend.service.type Frontend service type
```

Comment:
Putting it here because I don't have space. The following values are missing in values.yaml

```
Key frontend.topologySpreadConstraints not found
Key ingress.certManager not found
Key ingress.certManager.clusterIssuer not found
Key ingress.certManager.commonName not found
Key ingress.certManager.httpIngressClass not found
Key kubeops.initContainers not found
```

Reply:
The following values are in values.yaml but in the template:

```
testImage.pullPolicy
dashboard.skipAvailablePackageDetails

```

Reply:
The value `testImage.pullPolicy` is not referenced directly, the helper `kubeapps.imagePullSecrets` includes `testImage`.

Regarding `dashboard.skipAvailablePackageDetails`, maybe @antgamdia knows if it is deprecated or accidentally not used.

Reply:
Just pending @antgamdia regarding `dashboard.skipAvailablePackageDetails` not being used.

`ingress.certManager` deprecated. Added missing `kubeops.initContainer` and `frontend.topologySpreadConstraints`.

Reply:
Good catch! We do use `skipAvailablePackageDetails` but, somehow, it wasn't present in the chart. This feature was added at https://github.com/kubeapps/kubeapps/pull/3799, but it seems it didn't get added to https://github.com/bitnami/charts/blob/master/bitnami/kubeapps/templates/dashboard/configmap.yaml.

Therefore, we should add in [this file](https://github.com/bitnami/charts/blob/master/bitnami/kubeapps/templates/dashboard/configmap.yaml):

````
"skipAvailablePackageDetails": {{ .Values.dashboard.skipAvailablePackageDetails }}
````

So that it renders: https://github.com/kubeapps/kubeapps/blob/main/dashboard/public/config.json#L17

Reply:
Line added here:
https://github.com/bitnami/charts/blob/cd9b2f4b1741248f8243c65f6703361879240623/bitnami/kubeapps/templates/dashboard/configmap.yaml#L81-L84